packages/api/src/endpoints/channel/HelixSnoozeNextAd.ts

```diff
@@ -0,0 +1,32 @@
+// import { Enumerable } from '@d-fischer/shared-utils';
+import { DataObject, rawDataSymbol, rtfm } from '@twurple/common';
+// import type { BaseApiClient } from '../../client/BaseApiClient';
+import type { HelixSnoozeNextAdData } from '../../interfaces/endpoints/channel.external';
+// import type { HelixUser } from '../user/HelixUser';
+
+/**
+ * Represents a broadcaster's ad schedule snooze event.
```

Comment:
This is not an event, this is a result of a request. The name of the class doesn't really scream that either; maybe add an additional `Result` suffix to it.